src/routes/components/components.json

```diff
@@ -2546,5 +2546,15 @@
 		"category": "Data Visualisation",
 		"tags": ["charts", "components and libraries"],
 		"stars": 0
+	},
+	{
+		"title": "svelte-brick-gallery",
+		"url": "https://github.com/anotherempty/svelte-brick-gallery",
+		"description": "A masonry-like image gallery component for svelte",
+		"npm": "svelte-brick-gallery",
+		"addedOn": "2022-05-19",
+		"category": "User Interaction",
+		"tags": ["image", "gallery", "masonry"],
```

Comment:
The tags `gallery` and `masonry` are new.

I'm not sure about the need to create new ones.
Any thought about it @kevmodrome @benmccann ?

Reply:
I would avoid it. They don't seem like they'll be heavily used